pro/src/pages/LostPassword/__specs__/LostPassword.spec.tsx

```diff
@@ -41,34 +41,10 @@ describe('src | components | pages | LostPassword', () => {
         screen.getByLabelText(/Adresse e-mail/),
         'coucou@example.com'
       )
-      await userEvent.click(screen.getByText(/Envoyer/))
+      await userEvent.click(screen.getByText(/Valider/))
 
       // he has been redirected to next step
       expect(screen.getByText(/Merci/)).toBeInTheDocument()
     })
   })
-
-  describe('when user arrive from his reset password link', () => {
```

Comment:
Pourquoi ce test a été retiré ?

Reply:
Il test (/Nouveau mot de passe/), (/Mot de passe changé !/), (/Se connecter/)) mais il n'y a pas tout cela sur mes deux pages modifiées par mon ticket. il me faut juste page d' accueil et /mot-de-passe-perdu, pas  les autres chose. Donc j'ai mal fait, stp?  it    ('should be able to sent his password', async () => {
      // given
      const url = '/mot-de-passe-perdu?token=ABC'

      // when
      renderLostPassword(url)

      // then
      // user can fill and submit new password
      await userEvent.type(
        screen.getByLabelText(/Nouveau mot de passe/),
        'MyN3wP4$$w0rd'
      )
      await userEvent.click(screen.getByText(/Envoyer/))

      // he has been redirected to final step
      expect(
        await screen.findByText(/Mot de passe changé !/)
      ).toBeInTheDocument()
      expect(screen.getByText(/Se connecter/)).toBeInTheDocument()
    })
  })
})

Reply:
Oui @TetyanaZhabchyk  il faut laisser ce test, même s'il n'a pas de rapport avec ce que tu as fait
J'ai l'impression qu'il passe bien si tu le remets

Reply:
C'est fait et passe. 
MArie, maintenant c'est bon?